code/datums/components/food_stuff.dm

```diff
@@ -0,0 +1,238 @@
+/datum/component/consume
```

Comment:
can you describe the purpose of this component a bit

Reply:
Done, added a comment that explains the thing it does.